lib/schema.js

```diff
@@ -446,6 +446,26 @@ module.exports = {
           }
         },
         {
+          "description": "Paginate over list members in the given mailing list",
+          "href": "/lists/{address}/members/pages",
+          "method": "GET",
+          "title": "page",
```

Comment:
To keep consistent with the existing API patterns I would say the title should be either `get` or `list`